test/e2e/scheduling/preemption.go

```diff
@@ -200,7 +203,9 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			gomega.Expect(livePod.DeletionTimestamp).To(gomega.BeNil())
 		}
 
-		framework.ExpectEqual(podPreempted, true)
+		if !podPreempted {
+			framework.Failf("expected 'notFound' as error, got instead %v", err)
```

Comment:
Wrong failure message?

Reply:
I had deducted this result due to code above it, at line 198 i.e : 
`podPreempted := (err != nil && apierrors.IsNotFound(err)) ||
			  (err == nil && preemptedPod.DeletionTimestamp != nil)`

Reply:
Am I correct?

Reply:
`podPreempted` is more than that "not found", there's also the `preemptedPod.DeletionTimestamp != nil` part.

I would say `"expected pod to be preempted, instead got pod %+v and error %v", preemptedPod, err`.